docs/ref/contrib/admin/javascript.txt

```diff
@@ -0,0 +1,74 @@
+
```

Comment:
The blank line was at the very beginning of file when you made the line note, i removed it,  but the line note continues scrolling down to each new blank line in the file.